ERCS/erc-7598.md

```diff
@@ -0,0 +1,2756 @@
+---
+eip: 7598
```

Comment:
```suggestion
eip: 7596
```
This PR appears to be a duplicate of #204.  Please close this duplicate.  
FYI, EIP/RIP/ERC numbers are assigned by editors and associates.

Reply:
Well understood. Many thanks. 
I will close this PR and update the ERC number as 7596.  

Regards.
Paul

Reply:
eip number in the head, as well as the file name have been changed as per the new assigned number.  
Please review whether it's compliance with the rules. 

Many thanks.
Paul

Reply:
Hi Andrew,  
Could you please approve this PR so as to incorporated it into the Repo?